src/test/java/org/example/RunTest.java

```diff
@@ -0,0 +1,13 @@
+package org.example;
+
+
+import cucumber.api.CucumberOptions;
+import cucumber.api.junit.Cucumber;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(features = ".",tags = "@test",format = {"pretty","html:target/Destination"})
+
+public class RunTest
+{
```

Comment:
Why do you have empty class?

Reply:
Run Test class for only annotation. However, this types of annotation always write out side the class.